indexer/locals.tf

```diff
@@ -148,15 +163,10 @@ locals {
             name : "BLOCKED_ADDRESSES",
             value : var.indexer_compliance_blocklist,
           },
-          {
-            name : "PG_POOL_MAX",
-            value : "2"
-          },
-          var.roundtable_ecs_environment_variables,
```

Comment:
Why remove this?

Reply:
Incorrect resolved a merge conflict. fixed.